extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/SchedulerService.java

```diff
@@ -85,8 +89,8 @@ public Job createJob( JobScheduleRequest scheduleRequest )
 
     // Used to determine if created by a RunInBackgroundCommand
     boolean runInBackground =
-        scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
-            && scheduleRequest.getCronJobTrigger() == null;
+      scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
+        && scheduleRequest.getCronJobTrigger() == null;
```

Comment:
Work item processing starts when the work item is scheduled. I think we want to be able to track its status through its entire lifecycle, not just the lifecycle as it pertains to the new code we have written.